EKF/ekf.h

```diff
@@ -105,6 +105,9 @@ class Ekf : public EstimatorInterface
 
 	void getFlowInnovRatio(float &flow_innov_ratio) const override;
 
+	void getFlowVelBody(float flow_vel_body[2]) const override;
```

Comment:
I would prefer to return a `Vector2f` and do the copyTo on the Firmware side. This way if we want to use these functions in unit tests, they are easier to handle.

Reply:
Ok, yes, it makes sense; I just copied the existing pattern

Reply:
@kamilritz Done